eng/tools/generator/config/validate/remoteValidator.go

```diff
@@ -23,16 +23,19 @@ type remoteValidator struct {
 
 func (v *remoteValidator) Validate(cfg config.Config) error {
 	var errResult error
+	invalidReadme := map[string]struct{}{}
```

Comment:
where do we use this `invalidReadme`?

Reply:
I suppose to collect the invalid requests, and then delete then in cfg.Track2Requests